lib/rack/utf8_sanitizer.rb

```diff
@@ -7,8 +7,12 @@ module Rack
   class UTF8Sanitizer
     StringIO = ::StringIO
 
-    def initialize(app)
+    # options[:sanitizable_content_types] Array
+    # options[:additional_content_types] Array
+    def initialize(app, options={})
       @app = app
+      @sanitizable_content_types = options[:sanitizable_content_types]
+      @sanitizable_content_types ||= SANITIZABLE_CONTENT_TYPES + (options[:additional_content_types] || [])
```

Comment:
It's not important in the context of this PR, but yes, `SANITIZABLE_CONTENT_TYPES` is semantically a set and there could be some minor performance benefit from switching to that as well.